sample/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <resources>
     <string name="app_name">Auth0 SDK Sample</string>
-    <string name="com_auth0_domain">lbalmaceda.auth0.com</string>
-    <string name="com_auth0_client_id">PX2vJhpALUNT66NHdCdD30XWqlIR4oEZ</string>
+    <string name="com_auth0_domain">poovamraj.eu.auth0.com</string>
+    <string name="com_auth0_client_id">swarLtLAd0aW55ajPUPzYPXjtB3UtZki</string>
```

Comment:
Maybe we should replace these with placeholders?

Reply:
This would be the cleanest and straight forward way @Widcket.

Considering this is a public client this shouldn't be an issue. We use this in auth0-spa-js as well - [Link](https://github.com/search?q=repo%3Aauth0%2Fauth0-spa-js%20brucke.auth0.com&type=code)